.muffet-filter/mfw

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
+# muffet-filter wrapper
+# ----------------------------------------------------------------------------
+PROJECT_ORG="bhamail"
+PROJECT_NAME="muffet-filter"
+PROJECT_URL="https://github.com/$PROJECT_ORG/$PROJECT_NAME/muffet-filter"
+#echo "project url: $PROJECT_URL"
+
+WRAPPER_FILENAME="mfw"
+#echo "wrapper filename: $WRAPPER_FILENAME"
+
+WRAPPER_DIR="./.muffet-filter"
+WRAPPER="$WRAPPER_DIR/$WRAPPER_FILENAME"
+#echo "wrapper: $WRAPPER"
+
+if test -f "$WRAPPER"; then
+  echo "using wrapper: $WRAPPER"
+else
+  echo "::error::Missing muffet-filter wrapper file $WRAPPER. Download here: $PROJECT_URL" && exit 1
+fi
+
+MUFFET_FILTER_BINARY="$WRAPPER_DIR/muffet-filter"
+if test -f "$MUFFET_FILTER_BINARY"; then
+  echo "using binary: $MUFFET_FILTER_BINARY"
+else
+  echo "::error::Missing muffet-filter binary file $MUFFET_FILTER_BINARY   Downloading..."
```

Comment:
Just noticed this. Should this be an error? Based on the CI result, this appears to cause an `exit 1`

Reply:
This is the branch where the binary file is downloaded. Not sure why it would be an error. It’s OK if the doesn’t yet exist locally. The script fetches it.